activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -566,6 +571,10 @@ def drop_table(table_name, options = {})
       #  add_column(:users, :picture, :binary, limit: 2.megabytes)
       #  # ALTER TABLE "users" ADD "picture" blob(2097152)
       #
+      #  add_column(:users, :role, index: true)
+      #  # ALTER TABLE "users" ADD "role" varchar
+      #  # CREATE INDEX users_role_index ON users(role)
```

Comment:
Is this true? Do we have a test for that?

Reply:
I'm sorry, I did not verify this. It works like this:

```ruby
change_table :users do |t|
  t.string :role, index: true
end
```

But it does  not work in the ```add_column``` method. I do need to review this PR again.

Reply:
I removed this documentation comment, as it was simply incorrect. I added a sentence to the guide to point this out.